cli/src/main/kotlin/commands/UploadResultToSw360Command.kt

```diff
@@ -0,0 +1,142 @@
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.file
+
+import kotlin.time.measureTimedValue
+
+import org.eclipse.sw360.antenna.api.service.ServiceFactory
+import org.eclipse.sw360.antenna.sw360.client.adapter.SW360Connection
+import org.eclipse.sw360.antenna.sw360.client.adapter.SW360ConnectionFactory
+import org.eclipse.sw360.antenna.sw360.client.config.SW360ClientConfig
+import org.eclipse.sw360.antenna.sw360.client.rest.resource.releases.SW360Release
+import org.eclipse.sw360.antenna.sw360.client.utils.SW360ClientException
+
+import org.ossreviewtoolkit.GlobalOptions
+import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.model.config.Sw360StorageConfiguration
+import org.ossreviewtoolkit.model.readValue
+import org.ossreviewtoolkit.utils.expandTilde
+import org.ossreviewtoolkit.utils.formatSizeInMib
+import org.ossreviewtoolkit.utils.log
+import org.ossreviewtoolkit.utils.perf
+
+class UploadResultToSw360Command : CliktCommand(
+    name = "upload-result-to-sw360",
+    help = "Upload an ORT result to SW360.",
+    epilog = "EXPERIMENTAL: The command is still in development and usage will likely change in the near future. The " +
+            "command expects that a Sw360Storage for the scanner is configured."
+) {
+    private val ortFile by option(
+        "--ort-file", "-i",
+        help = "The ORT result file to read as input."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+        .inputGroup()
+
+    private val globalOptionsForSubcommands by requireObject<GlobalOptions>()
+
+    override fun run() {
+        val (ortResult, duration) = measureTimedValue { ortFile.readValue<OrtResult>() }
+
+        log.perf {
+            "Read ORT result from '${ortFile.name}' (${ortFile.formatSizeInMib}) in ${duration.inMilliseconds}ms."
+        }
+
+        val sw360Config = globalOptionsForSubcommands.config.scanner?.storages?.values
+            ?.filterIsInstance<Sw360StorageConfiguration>()?.singleOrNull()
+
+        requireNotNull(sw360Config) {
+            "No SW360 storage is configured for the scanner."
+        }
+
+        createSw360Connection(sw360Config)?.let { sw360Connection ->
+            val sw360ReleaseClient = sw360Connection.releaseAdapter
+
+            ortResult.getPackages(omitExcluded = true).map { (pkg, _) ->
+                try {
+                    val release = SW360Release()
+                        .setMainLicenseIds(pkg.declaredLicenses)
+                        .setName(pkg.id.name)
+                        .setVersion(pkg.id.version)
+
+                    sw360ReleaseClient.createRelease(release)?.let {
+                        log.debug { "Release ${release.name}-${release.version} created in SW360." }
+                    }
```

Comment:
This is okay for an initial PoC. In the future, we need to check whether the release already exists in SW360 and only create it if not.

In addition, we will probably have to create a project in SW360 representing the current ORT project and link releases to it. The information which project uses which releases is going to be needed.

Reply:
This then at least demands a `TODO` in the code. Better yet, just implement it right now.